test/test_automl/test_automl.py

```diff
@@ -573,28 +574,3 @@ def test_fail_if_feat_type_on_pandas_input(backend, dask_client):
             task=BINARY_CLASSIFICATION,
             feat_type=['Categorical', 'Numerical'],
         )
-
-
-def test_fail_if_dtype_changes_automl(backend, dask_client):
-    """We do not support changes in the input type.
-    Once a estimator is fitted, it should not change data type
-    """
-    automl = autosklearn.automl.AutoML(
-        backend=backend,
-        time_left_for_this_task=30,
-        per_run_time_limit=5,
-        metric=accuracy,
-        dask_client=dask_client,
-    )
-
-    X_train = pd.DataFrame({'a': [1, 1], 'c': [1, 2]})
-    y_train = [1, 0]
-    automl.InputValidator.validate(X_train, y_train, is_classification=True)
-    with pytest.raises(
-        ValueError,
-        match="Auto-sklearn previously received features of type"
```

Comment:
This is no longer tested for as it now is a warning, correct?

Reply:
Yes!